tests/zfs-tests/tests/functional/cli_root/zpool_expand/zpool_expand_003_neg.ksh

```diff
@@ -27,23 +27,23 @@
 
```

Comment:
So, we're both confused! :)  So it turns out there are 2 types of autoexpand. The zpool_expand tests 001 and 003 are only testing one case AFAIKT. One case is where an existing LUN gets bigger and the other case is where a smaller LUN is replaced by a bigger one (that is what the spa is checking).  I'm thinking we can have a 004 neg that is the opt-out test for the 002 test (ie the one with files).